std/experimental/allocator/building_blocks/package.d

```diff
@@ -252,11 +252,11 @@ simple bump-the-pointer allocator.))
 $(TR $(TDC2 InSituRegion, region) $(TD Region holding its own allocation, most often on
 the stack. Has statically-determined size.))
 
-$(TR $(TDC2 SbrkRegion, region) $(TD Region using $(D $(LINK2 https://en.wikipedia.org/wiki/Sbrk,
-sbrk)) for allocating memory.))
+$(TR $(TDC2 SbrkRegion, region) $(TD Region using $(LINK2 https://en.wikipedia.org/wiki/Sbrk,
+`sbrk`) for allocating memory.))
 
 $(TR $(TDC3 MmapAllocator, mmap_allocator) $(TD Allocator using
-            $(D $(LINK2 https://en.wikipedia.org/wiki/Mmap, mmap)) directly.))
+            $(LINK2 https://en.wikipedia.org/wiki/Mmap, `mmap`) directly.))
```

Comment:
NB: the extra added space

![image](https://cloud.githubusercontent.com/assets/4370550/23589594/12c074bc-01d0-11e7-9293-f3e80e747738.png)